GitUI/GitUI.csproj

```diff
@@ -24,6 +24,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="AppInsights.WindowsDesktop" Version="$(AppInsightsWindowsDesktopVersion)" />
+    <PackageReference Include="ExCSS" Version="2.0.6" />
```

Comment:
@NikolayXHD what was the reason not to use the [latest version 3.0.0](https://www.nuget.org/packages/ExCSS/3.0.0) that supports .NET Standard?
I've tried to update the app the .NET 5.0 and this library caused quite a bit of failures...

Reply:
This PR description says v3 is unusable, https://github.com/TylerBrinks/ExCSS/issues/94

Reply:
ExCSS 3 is unfinished work in many aspects. ExCSS_Core does complete some things, but not all. I gave up on it when it became clear, that color values are not parsed, so while 
v2 produces ready-to-use Color structure
v3 produces a plain text 
```
"#ffffff", "hsl(1.0, ...)", ...
```

Reply:
Had another look at ExCSS-Core now v4, error handling is also incomplete. Parsing errors are silently ignored, there is an event raised when error happens, but no way to subscribe to it using public API

https://github.com/HockeyJustin/ExCSS_Core/blob/9a1114494d87220f38e7a157e3307ae6d4e4efb3/src/ExCSS/Parser/Lexer.cs#L8

![image](https://user-images.githubusercontent.com/12046452/95141010-cb1bd180-0778-11eb-928b-6168b70e31a0.png)

Reply:
If ExCSS v2 is problematic to use, I believe I can still rollback to manual parsing, since we actually use a very small subset of CSS.

Reply:
Now it feels quite nice to have theme parsing decently covered by tests 😃

Reply:
Oh bugger!

I spent a number of very frustrating hours yesterday trying to update my port to .NET 5.0, and ExCss turned out to be a major hurdle. I commented it out, but since we moved all colour handling to it, I couldn't get any colours working... 😞 

https://github.com/gitextensions/gitextensions/pull/8522

Reply:
> I believe I can still rollback to manual parsing, since we actually use a very small subset of CSS.

When you have time, may be you could work directly off my branch?
Much appreciate it, thank you

Reply:
Hmm, may be we could even fork ExCss 2.x and compile as .NET Standard?

Reply:
If we are OK to fork, then I'd fork ExCSS-Core and fix the things we miss. Later submit a PR, get it merged and replace back the fork with nuget package.

Reply:
:smirk: (imagening hsl-defined colors in theme .css)

Reply:
There - https://github.com/gitextensions/ExCSS, go for your life 😉

Reply:
We should probably fork from another fork https://github.com/HockeyJustin/ExCSS_Core
because the original repo seems abandoned, they won't accept any PRs

https://github.com/TylerBrinks/ExCSS/issues/99
https://github.com/TylerBrinks/ExCSS/issues/102

Reply:
Apologies, I should read more thoroughly...
Here: https://github.com/gitextensions/ExCSS_Core

Reply:
Hey, I just found the parser used by Visual Studion and ASP.NET - https://github.com/dotnet/cssparser. Maybe worth looking at it too?

Reply:
absolutely will have a look at it

Reply:
It's only going to happen the weekend though, struggling at work.

Reply:
Absolutely no pressure, there's a major issue (https://github.com/gitextensions/gitextensions/pull/7162#issuecomment-674834068) in the editor that blocks the migration anyway.

Reply:
I like the text in the Readme.md:

>This parser is an internal implementation detail of the .NET Core SDK, and is not supported for any other usage. We do not recommend taking a dependency on this code because there is no guarantee of long-term maintenance.

Reply:
Haha, yes I was told there's no official ownership of this repo.
But if it's good for big guys, it is _probably_ good for us. And we can fork it, if necessary.

Reply:
ExCSS core is now resurrected, v4 has been released.

Reply:
I've looked at it some time ago, It's still pretty much unusable.

> Had another look at ExCSS-Core now v4, Concrete rule types which provide access to specific parsed values are all internal
https://github.com/HockeyJustin/ExCSS_Core/blob/master/src/ExCSS/Rules/ImportRule.cs